src/util/ringct.rs

```diff
@@ -348,6 +389,19 @@ pub enum RctType {
     Bulletproof2,
 }
 
+impl Display for RctType {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), FmtError> {
+        let rct_type = match self {
+            RctType::Null => "Null",
+            RctType::Full => "Full",
+            RctType::Simple => "Simple",
+            RctType::Bulletproof => "Bulletproof",
+            RctType::Bulletproof2 => "Bulletproof2",
+        };
+        writeln!(fmt, "{}", rct_type)
```

Comment:
I think we can use `write!` instead of printing a new line here

Reply:
Done